### PR TITLE
chore(sdk): update the cc types and scopes for simplicity

### PR DESCRIPTION
**Pull request by kptkin**
Description
-----------

What does the PR do? Include a concise description of the PR contents.

- [x] I updated CHANGELOG.md, or it's not applicable

Testing
-------
How was this PR tested?

### CODE REVIEW

File: .github/workflows/check-pr-title.yml [hunk 1]
```diff
@@ -11,38 +11,30 @@ jobs:
...
-      - uses: amannn/action-semantic-pull-request@v5.5.2
+      - uses: amannn/action-semantic-pull-request@v5.5.3
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
         with:
-          # Allowed types: add a notion doc
           types: |
             feat
             fix
-            docs
```
Comment (dmitryduev) [inline .github/workflows/check-pr-title.yml:22]: let's keep

File: .github/workflows/check-pr-title.yml [hunk 2]
```diff
@@ -11,38 +11,30 @@ jobs:
...
             refactor
-            perf
-            test
-            build
-            ci
+            style
             chore
             revert
             security
             release
```
Comment (dmitryduev) [inline .github/workflows/check-pr-title.yml:26]: rm?

File: .github/workflows/check-pr-title.yml [hunk 3]
```diff
@@ -11,38 +11,30 @@ jobs:
...
             artifacts
-            media
-            sweeps
-            launch
             integrations
-            deps
+            launch
+            sdk
+            sweeps
           requireScope: true
```
Comment (dmitryduev) [inline .github/workflows/check-pr-title.yml:34]: false?

**Review by dmitryduev — APPROVED** [review]

File: .github/workflows/check-pr-title.yml (hunk 3 above)
Comment (kptkin) [reply .github/workflows/check-pr-title.yml:34]: yeah, i was debating this...

File: .github/workflows/check-pr-title.yml (hunk 1 above)
Comment (kptkin) [reply .github/workflows/check-pr-title.yml:22]: i thought about combining it with style but open to keeping it... trying to reduce the number of types so it is easier to remember and/or map

File: .github/workflows/check-pr-title.yml (hunk 2 above)
Comment (kptkin) [reply .github/workflows/check-pr-title.yml:26]: oh just use chore?

**Pull request merged**